packages/nextjs-mf/src/plugins/container/runtimePlugin.ts

```diff
@@ -2,6 +2,19 @@ import { FederationRuntimePlugin } from '@module-federation/runtime/types';
 export default function (): FederationRuntimePlugin {
   return {
     name: 'next-internal-plugin',
+    //@ts-ignore
+    createScript({ url, attrs }) {
+      if (typeof window !== 'undefined') {
+        const script = document.createElement('script');
+        script.src = url;
+        script.async = true;
+        //@ts-ignore
+        delete attrs.crossorigin;
```

Comment:
prevent cors error in next till we can find better solution to enable/disable cross origin